### DISCUSSION TITLE
Number mask scale range

### DISCUSSION THREAD

**Discussion by localusercamp** [uNmAnNeR/imaskjs#819] [Q&A]
Hi!

Is there any way to mask scale range?

vue-currency-input has same option as scale - precision. In docs they have:
![image](https://user-images.githubusercontent.com/45385480/215015583-8580e6b6-9bb8-4b03-a245-4a0af7cf991d.png)

For example if we have `scale = { min: 2, max: 4 }`, then if number has 1 or 2 digits after radix it will display as `,##`, but if you continue to input new digits after radix 3 or 4 for example it will be shown as `,####`.

**Comment by uNmAnNeR** [comment]
can be done via dynamic mask
